bin/git-test

```diff
@@ -473,6 +473,7 @@ class Test(object):
             if retcode:
                 if verbosity == level - 1:
                     sys.stderr.write(out)
+                    sys.stderr.write(err)
```

Comment:
I would expect the line

    kwargs = dict(stdin=subprocess.PIPE, stderr=subprocess.STDOUT)

(a few lines higher) to cause stderr to be written to stdout and therefore already be output in the error case (and, conveniently, it should be chronologically intermingled with the stdout). Is that not working?

Reply:
This was initially triggered by an unused variable warning on the `err` variable, but rather than remove it I though why not dump that as well in addition to stdout, hence the added line. But as you noticed I did not study the surrounding code that thoroughly, and you are correct in that that the line is pointless. I'll update the branch.